resources/js/src/app/components/customer/AddressSelect/CreateUpdateAddress/CreateUpdateAddress.js

```diff
@@ -133,11 +127,30 @@ Vue.component("create-update-address", {
                     this.$dispatch("new-address-created", this.addressData);
 
                     this.waiting = false;
-                }.bind(this))
-                .fail(function()
+                })
+                .fail(response =>
                 {
                     this.waiting = false;
-                }.bind(this));
+
+                    if (response.validation_errors)
+                    {
+                        this._handleValidationErrors(response.validation_errors);
+                    }
+                });
+        },
+
+        _handleValidationErrors(validationErrors)
+        {
+            ValidationService.markFailedValidationFields($(this.addressFormNames[this.addressType]), validationErrors);
+
+            let errorMessage = "";
+
+            for (const i in validationErrors)
```

Comment:
The for each...in statement is deprecated as the part of ECMA-357 (E4X) standard. Aber for... of. .